src/llmq/quorums_utils.cpp

```diff
@@ -44,13 +44,20 @@ uint256 CLLMQUtils::BuildSignHash(Consensus::LLMQType llmqType, const uint256& q
     return h.GetHash();
 }
 
-std::set<uint256> CLLMQUtils::GetQuorumConnections(Consensus::LLMQType llmqType, const CBlockIndex* pindexQuorum, const uint256& forMember)
+std::set<uint256> CLLMQUtils::GetQuorumConnections(Consensus::LLMQType llmqType, const CBlockIndex* pindexQuorum, const uint256& forMember, bool onlyOutbound)
 {
     auto& params = Params().GetConsensus().llmqs.at(llmqType);
 
     auto mns = GetAllQuorumMembers(llmqType, pindexQuorum);
     std::set<uint256> result;
 
+    if (!onlyOutbound) {
+        for (auto& dmn : mns) {
+            result.emplace(dmn->proTxHash);
+        }
+        return result;
+    }
+
```

Comment:
This will return incorrect data when spork21 is off, pls see my previous suggestion for the code changes I think would be more appropriate.

Reply:
Ahh, now I understand. I used your suggestion in the spork21 case and also changed the !spork21 case to mimic the same behavior (this required some small refactoring).